js/reducers/user.js

```diff
@@ -21,6 +22,7 @@ import config from '../config'
 // user state when logged out
 export type LoggedOutUserState = {
   isLoggedIn: false,
+  loginError?: boolean,
```

Comment:
boolean attributes/variables should be named `isSomething` or `hasSomething`, in this case `isError`

said that, I'm not sure what it does mean, please add a short comment describing what it means when that attribute is `true` - also, perhaps, worth considering splitting the `LoggedOutUserState` type in two: `DefaultLoggedOutUserState` and `FailedLoggedOutUserState`

Reply:
fixed